alumet/src/agent.rs

```diff
@@ -807,6 +807,18 @@ mod tests {
 
     use super::{AgentConfig, ENV_VAR};
 
+    #[test]
+    fn static_plugins_macro() {
```

Comment:
Maybe you could test using:
```rust
 let x = static_plugins![,];
```

or is it not allowed ?

Reply:
good idea